client/src/templates/Challenges/classic/classic.css

```diff
@@ -142,5 +142,5 @@
 }
 
 .nav-tabs [role='tab']:focus:not(:focus-visible) {
-  outline: none;
+  outline-color: transparent;
```

Comment:
```suggestion
```

Revert back to outline: none. This is for non-keyboard focus in which we don't want a focus outline.